index.js

```diff
@@ -11,6 +28,22 @@ module.exports = {
     };
   },
 
+  config: function(/* environment, appConfig */) {
+    // do nothing unless running `ember fastboot` command
+    if (!process.env.EMBER_CLI_FASTBOOT) { return {}; }
+
+    return {
+      autoRun: false,
+      storeConfigInMeta: false,
+      EmberENV: {
+        FEATURES: { 'ember-application-visit': true }
```

Comment:
I initially avoided doing this because feature flags were not being merged.  Which meant that if you had a feature flag set in the consuming apps `config/environment.js` it would completely ignore any features added from an addon's `config` hook.

This may be fixed now with current ember-cli, but I'd love it if you could confirm.  The easiest way to test would be to make a throw away app, set some features in it, run `ember fastboot` and confirm that the values are merged propertly in the resulting files...